cpp/models/abm/README.md

```diff
@@ -1,57 +1,127 @@
+5. Simulation: runs the simulation and stores results.
 
 ## Simulation
 
-The simulation runs in discrete time steps. Each step is in two phases, an interaction phase and a migration phase. 
+The simulation runs in discrete time steps. Each step has two phases, an interaction phase and a migration phase.
+
+### Interaction Phase
+
+In this phase each person interacts with the other persons at the same location. This interaction determines the transmission of the desease. A susceptible person can become infected by contact with an infected person. The probability of infection depends on a multitude of factors, such as the viral load and infectiousness of the infectee and the immunity level of the susceptible person.
 
-First each person interacts with the other persons at the same location. This interaction determines their infection state transitions. A susceptible person may become exposed by contact with an infected person without symptoms. The probability of infection depends on a multitude of factors, such as the viral load and infectiousness of the infectee and the immunity level of the susceptible person.
+### Migration Phase
 
 During the migration phase, each person may change location. Migration follows complex rules, taking into account the current location, time of day, and properties of the person (e.g. age). Some location changes are deterministic and regular (e.g. going to work), others are random (e.g. going to shopping or to a social event in the evening/on the weekend).
 
-The result of the simulation is for each time step the count of persons in each infection state at that time.
+Another way of migration we use in the simulation of Braunschweig (simulations/abm_braunschweig.cpp) is using trips. A trip consists of the ID of the person that performs this trip, a timepoint when this trip is performed and where the person is heading to. In the beginning of the simulation a list with all trips is initialized and followed during the simulation. There can be different trips on the weekend than during the week but other than that the agents do the same trips every day assuming they are not in quarantine or in the hospital.
+
+## Get Started
+
+This section gives an introduction how to use the ABM and setup your own simulation. For a quick overview can find a full example in the [abm minimal example](../../examples/abm_minimal.cpp) and a more detailed doxygen documentation [here](https://dlr-sc.github.io/memilio/documentation/index.html). For a guide on installation and running the simulations and examples see this [README](../../README.md).
 
-## Example
-An example can also be found at simulations/abm.cpp
+Every person in the ABM belongs to an AgeGroup which we can define:
+
+```cpp
+size_t NUM_AGE_GROUPS         = 4;
+const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(NUM_AGE_GROUPS - 4);
```

Comment:
This 'NUM_AGE_GROUPS - n' still confuses me, why not simply write the number? I would argue that it is an unnecessary complication, and inconsistent with other comments or code that use e.g. `AgeGroup(0)`. It would also make this example easier.